login-workflow/docs/customization.md

```diff
@@ -0,0 +1,86 @@
+# Customization Guide
+While the default workflow is designed to work for most applications, we understand that you may need to customize the workflow to meet your needs. This guide will walk you through the various ways you can customize the workflow.
+
+## Customizing Registration Workflow
+
+### Customizing the Screen Order
+You may want to customize the order of your screens within the workflow. In order to do this you will need to import the screens you want to use and pass them as children to the `RegistrationWorkflow` component. The screens will render in the order they are passed in.
+
+Our default implementation looks like this:
+  
+```jsx
+<RegistrationWorkflow initialScreenIndex={0}>
```

Comment:
In default workflow, we don't need to pass initialScreenIndex={0}, as it is set to 0 by default.